campaigns/rustreach/how-I-came-to-rust.md

```diff
@@ -0,0 +1,58 @@
+* No single-person "owner" of Rust (BDFL): I like that the Rust project is an ongoing conversation. From language features, over community action to where we want to apply our power. And it somehow works.
+* The mutual understanding that issues will always exist. We're a global project of mostly amateurs. We might do similar things in our day-job, but most of us are _not_ working on a global async project for free as their main profession. The list of problems we had in the last three years is interesting and many, I have never faced before.
+* Rust has working complaint structures. If I have a problem with how things are going, I know where to send them. I know someone will reply with appropriate urgency. (A week or so, we are hobbyists)
+* Issues, especially social ones, will be dealt with. At some point, obviously, but we work on things. I like the way the project is improving at a steady pace and not rushing things.
+* Actively working for a good tone and balance. Rust has people of many cultures and communication issues arise from that. I don't want to make that sound like I'm the master of this, I have been called out multiple times for ways of arguing or tone. This is great, it starts a conversation and allows to check behaviour.
+* No "you must be this tall to be a project member". If you want to start working on something of project capacity, let us know!
+* We appreciate that people have lives outside of the project. You can only spend 4 hours a month? Cool, let's make sure they count!
+
+A friend once told me that I see problem everywhere. And she's right. But a stated problem is something you can work on. The Rust project, over the last years, has found a good working scheme in which problems are stated, inspected and subsequently fixed. I like that. And for me personally, that means my problems are taken seriously. My part of the trade is that I take all of the other problems with seriousness and allow them to have room.
+
+## Taking part
+
+When you start #rustreach, you'll be taken into a project that is written in Rust. For that, you will mainly interact with your mentor and the library or program you will be working on. Sometimes, that means that the Rust project - the organisation building this programming language - will mostly be a secondary concern.
+
+I would still like to encourage you to check us out a little. While there's no "formal" way to join most of the teams, it's mostly easy: show up and be willing to work on tasks. Or, even better, if you have a subject that is not covered by any of the groups, form one! It isn't even _as_ huge of an amount of work as you may think.
+
+The barriers are much lower then you might think and tasks are aplenty!
```

Comment:
Great post! I hope I didn't put you off too much with my comments above. It looks very good to me. Cheers!